main.go

```diff
@@ -45,4 +55,15 @@ func main() {
 	close(swapIntentOperationBroadcastChan)
 	close(solverOperationSubmitChan)
 	close(governanceSignatureChan)
+
+	// Questions:
+	// entities:
+	//    backend - is this the bundler that gets the intent, distribute to solvers, collect solutio and create the metaTransactions? you can it backend and not bundler as there can be other configurations?
+	//    governance - what is this entity? is it part of the dApp or part of Atlas?
+	//    user - represents a user application or a real user + dApp ui?
```

Comment:
In this example, the user builds its intent and signs its user operation. This would need to be handled by the dApp UI, as we don't expect the user to manually do those steps.